packages/runtime/src/lib/txn.ts

```diff
@@ -57,6 +57,13 @@ export function parseToStackElem(a: unknown, field: TxField): StackElem {
 	if (Buffer.isBuffer(a)) {
 		return new Uint8Array(a);
 	}
+
+	// case Uint8Array but empty, some magic here
+	// I also don't how it work :|
+	if (ArrayBuffer.isView(a)) {
```

Comment:
if a === [] then Buffer.isBuffer will return false.

Reply:
`ArrayBuffer.isView([]) === false`
So maybe we should check `Array.isArray`? and put it in one if:
```
if (Buffer.isBuffer(a) || Array.isArray(a) || ArrayBuffer.isView(a))
```

Reply:
Yup! I will try update it.